presage-cli/src/main.rs

```diff
@@ -90,6 +90,22 @@ enum Cmd {
         )]
         device_name: String,
     },
+    #[clap(
+        about = "Add a new secondary device to this (primary) device via URL (see link-device)"
+    )]
+    AddDevice {
+        #[clap(
+            long,
+            short = 'u',
+            help = "The URL (that is represented as QR code) created by the secondary device (see link-device)"
+        )]
+        uri: String,
```

Comment:
you should be able to use the `Uri` type directly IIRC

Reply:
I'll check, I also need to force push since my rebase or squash somehow added those old commits (which are already on main)

Reply:
Thx for the hint, I changed it to url directly